packages/rn-tester/package.json

```diff
@@ -1,5 +1,6 @@
 {
-  "name": "@react-native/tester",
+  "private": true,
+  "name": "@react-native/rn-tester",
```

Comment:
Hey @tido64,
Can I ask you to undo the renaming?

We had some discussions internally about this specific package (RN Tester) and it turns out that the status quo is the preferred way.

- `@react-native/tester` is the correct name of this package as it's the "React Native Tester"
- `packages/rn-tester` we're going to keep the folder name as it has better discoverability + we're depriotizing the folder rename (as it brings little value at the moment and it slow us down drastically). 

More on this here: https://github.com/facebook/react-native/issues/34692

The other changes are valid